src/Pos/Binary/Txp.hs

```diff
@@ -68,13 +68,24 @@ instance Bi T.TxDistribution where
                     nonEmpty $ replicate n []
                 Right ds -> pure ds
 
+instance Bi T.TxSigData where
+    put T.TxSigData{..} = do
```

Comment:
Why not using `Monoid` here? Also, did someone benchmarked difference so I stop worrying about that :)